jetty-home/pom.xml

```diff
@@ -95,30 +95,12 @@
               <artifactItems>
                 <artifactItem>
                   <groupId>org.eclipse.jetty.toolchain.setuid</groupId>
-                  <artifactId>jetty-setuid-java</artifactId>
+                  <artifactId>jetty-setuid-jna</artifactId>
```

Comment:
Could you please confirm that `jna-5.13.0.jar` will also be copied to `${assembly-directory}/lib/setuid`?